### PR TITLE
Add an Animation Resource

### LINKED ISSUE #982: [Add] Animation Resource
### Site URL

https://animista.net/

### Description of the above provided site

Animista is a CSS animation library and a place where you can play with a collection of ready-made CSS animations and download only those you will use.

### Category

Frontend

### Sub Category

animations

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Sriparno08**
## Changes proposed

I have added an animation resource to the website.

## Screenshots

Screenshot 2023-06-11 162710

## Note to reviewers

This PR is part of GSSoC'23.

### CODE REVIEW

File: database/frontend/animations.json [hunk 1]
```diff
@@ -81,5 +81,12 @@
         "url": "https://nzbin.github.io/three-dots/",
         "category": "frontend",
         "subcategory": "animations"
+    },
+    {
+        "name": "Animista",
+        "description": "Animista is a CSS animation library and a place where you can play with a collection of ready-made CSS animations and download only those you will use.",
```
Comment (CBID2) [inline database/frontend/animations.json:87]: [Proofreading]
```suggestion
        "description": "Animista is a library where you can play with a collection of ready-made CSS animations.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Sriparno08! :) I gave a suggestion on how to improve your description! :)

File: database/frontend/animations.json (hunk 1 above)
Comment (Sriparno08) [reply database/frontend/animations.json:87]: Okay, let me fix it.

**Comment by Sriparno08** [comment]
Hey @CBID2, I have updated it. Have a look!

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by ujjawaltyagii — APPROVED** [review]

**Review by rupali-codes — DISMISSED** [review]

**Comment by rupali-codes** [comment]
@Sriparno08 could you please resolve conflicts?

**Comment by Sriparno08** [comment]
Fixed, @rupali-codes!

**Review by rupali-codes — APPROVED** [review]
thanks

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Pull request merged**